content/docs/user-guide/experiment-management/checkpoints.md

```diff
@@ -85,6 +85,9 @@ $ dvc stage add --name train --deps data/MNIST --deps train.py \
               --params seed,lr,weight_decay --live dvclive python train.py
 ```
 
+The `--live dvclive` option enables our special logger [DVCLive](/doc/dvclive),
```

Comment:
Depending on https://github.com/iterative/dvclive/issues/60#issuecomment-848593068 we may want to end up removing `--live dvclive` from the example. Just something to keep in mind for later cc @dberenbaum and @flippedcoder